package.json

```diff
@@ -24,7 +25,10 @@
     "version:latest": "yarn changeset version && yarn angular build",
     "publish:latest": "yarn changeset publish",
     "canary:build": "yarn --cwd canary/ build",
-    "canary:install": "yarn --cwd canary/ install "
+    "canary:install": "yarn --cwd canary/ install ",
+    "canary:clean": "yarn --cwd canary clean ",
+    "canary:updatePackage": "node script/changeCanaryPackageVersion.js",
```

Comment:
Could we just change all the canary apps to `next` instead of running this script every time we do a release?

Reply:
Good point. I haven't looked into out canary yet and I just wrote the script basing on the preflight step in the runbook. 

So I'd like to know how @wlee221 would think

Reply:
That makes sense! 👍

Reply:
I was just thinking maybe we want to change the canaries to always run on next. But probably need to talk it over more with @wlee221 and others before we do that.

Reply:
IMO our canaries should still run against `latest`, because that's what the customers are installing in their apps. We should test what customers are install.

`next` makes more sense in release context, in which we want to test what we're pushing onto latest.

Reply:
@0618 Can you change the canaries to use `latest` instead of what they are on and include it in this PR? Since you are already updating some of the `package.json` files.